### PR TITLE
Changelog:Fix err hyperlinks  about changelog 1.20

### PR DESCRIPTION
**Pull request by yangjunmyfm192085**
Signed-off-by: JunYang <yang.jun22@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
1) Change the link type, from http to https
2) pr add author
3) sig-cli add  hyperlink

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

**Review by wilsonehusin — COMMENTED** [review]
hey I'm not sure if we should merge this one...

File: CHANGELOG/CHANGELOG-1.20.md [hunk 1]
```diff
@@ -479,7 +479,7 @@ Troubleshoot distroless containers by adding a new container with debugging tool
 Troubleshoot on a node by creating a container running in the host namespaces and with access to the host’s filesystem.
 Note that as a new builtin command, `kubectl debug` takes priority over any `kubectl` plugin named “debug”. You will need to rename the affected plugin.
 Invocations using `kubectl alpha debug` are now deprecated and will be removed in a subsequent release. Update your scripts to use `kubectl debug` instead of `kubectl alpha debug`!
-For more information about kubectl debug, see Debugging Running Pods on the Kubernetes website, kubectl help debug, or reach out to SIG CLI by visiting #sig-cli or commenting on [enhancement #1441](https://features.k8s.io/1441).
+For more information about kubectl debug, see Debugging Running Pods on the Kubernetes website, kubectl help debug, or reach out to SIG CLI by visiting [#sig-cli](https://github.com/kubernetes/community/tree/master/sig-cli) or commenting on enhancement [#1441](https://github.com/kubernetes/enhancements/issues/1441).
```
[the remark this answers is not in the document]
Comment (yangjunmyfm192085) [reply CHANGELOG/CHANGELOG-1.20.md]: I  agree,and fix it .

File: CHANGELOG/CHANGELOG-1.20.md [hunk 2]
```diff
@@ -504,7 +504,7 @@ Kubernetes will no longer ship an instance of the Cloud Controller Manager binar
 ## Known Issues
 
 ### Summary API in kubelet doesn't have accelerator metrics
-Currently, cadvisor_stats_provider provides AcceleratorStats but cri_stats_provider does not. As a result, when using cri_stats_provider, kubelet's Summary API does not have accelerator metrics. [#96873](https://github.com/kubernetes/kubernetes/pull/96873).
+Currently, cadvisor_stats_provider provides AcceleratorStats but cri_stats_provider does not. As a result, when using cri_stats_provider, kubelet's Summary API does not have accelerator metrics. ([#96873](https://github.com/kubernetes/kubernetes/pull/96873),[@ruiwen-zhao](https://github.com/ruiwen-zhao)).
```
[the remark this answers is not in the document]
Comment (yangjunmyfm192085) [reply CHANGELOG/CHANGELOG-1.20.md:625]: Is it better to be consistent with other places?

File: CHANGELOG/CHANGELOG-1.20.md [hunk 3]
```diff
@@ -1364,7 +1364,7 @@ filename | sha512 hash
   - Apply toleration for a new, future taint "node-role.kubernetes.io/control-plane:NoSchedule" to the kubeadm CoreDNS / kube-dns managed manifests. Note that this taint is not yet applied to kubeadm control-plane nodes.
   - Please adapt your workloads to tolerate the same future taint preemptively.
   
-  For more details see: http://git.k8s.io/enhancements/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint/README.md ([#95382](https://github.com/kubernetes/kubernetes/pull/95382), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]
+  For more details see: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint/README.md ([#95382](https://github.com/kubernetes/kubernetes/pull/95382), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]
```
[the remark this answers is not in the document]
Comment (yangjunmyfm192085) [reply CHANGELOG/CHANGELOG-1.20.md:1485]: I think https is better than http, do you think so ?
There is  already  no http link in other changelogs.

File: CHANGELOG/CHANGELOG-1.20.md [hunk 4]
```diff
@@ -516,7 +516,7 @@ Currently, cadvisor_stats_provider provides AcceleratorStats but cri_stats_provi
...
 - For CSI drivers, kubelet no longer creates the target_path for NodePublishVolume in accordance with the CSI spec. Kubelet also no longer checks if staging and target paths are mounts or corrupted. CSI drivers need to be idempotent and do any necessary mount verification. ([#88759](https://github.com/kubernetes/kubernetes/pull/88759), [@andyzhangx](https://github.com/andyzhangx)) [SIG Storage]
-- Kubeadm: http://git.k8s.io/enhancements/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint/README.md ([#95382](https://github.com/kubernetes/kubernetes/pull/95382), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]
+- Kubeadm: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint/README.md ([#95382](https://github.com/kubernetes/kubernetes/pull/95382), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]
```
[the remark this answers is not in the document]
Comment (yangjunmyfm192085) [reply CHANGELOG/CHANGELOG-1.20.md:637]: I think https is better than http, do you think so ?
There is  already  no http link in other changelogs.

File: CHANGELOG/CHANGELOG-1.20.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yangjunmyfm192085) [reply CHANGELOG/CHANGELOG-1.20.md:625]: I hope to actively participate in the k8s project very much.
I like this project very much and I am working on it.
Can i get your help?
Thank you very much.

**Comment by yangjunmyfm192085** [comment]
> hey I'm not sure if we should merge this one...

Ok,In response to the following changes, I have made an explanation.
Could you think about it?

**Comment by yangjunmyfm192085** [comment]
> hey I'm not sure if we should merge this one...

Okay, I made an explanation for the following changes.
Could you think about it?

**Comment by wilsonehusin** [comment]
> I hope to actively participate in the k8s project very much.

@yangjunmyfm192085 work in Kubernetes is usually planned and defined through issues, so PRs without an Issue which has been agreed upon would be hard for maintainers to manage since they would show up unplanned.

[I recommend looking into `good first issue` label and volunteer yourself to work on those](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

Happy to chat more on [Kubernetes Slack](https://slack.k8s.io) if you'd like to be involved in release notes work, I'm `@wilsonehusin` and you can say hi to `#release-notes` channel.

**Comment by yangjunmyfm192085** [comment]
> > I hope to actively participate in the k8s project very much.
> 
> @yangjunmyfm192085 work in Kubernetes is usually planned and defined through issues, so PRs without an Issue which has been agreed upon would be hard for maintainers to manage since they would show up unplanned.
> 
> [I recommend looking into `good first issue` label and volunteer yourself to work on those](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
> 
> Happy to chat more on [Kubernetes Slack](https://slack.k8s.io) if you'd like to be involved in release notes work, I'm `@wilsonehusin` and you can say hi to `#release-notes` channel.

Thank you very much,I will continue to learn more about  `good first issue`

**Comment by wilsonehusin** [comment]
thanks! see you around 😄 

/close

**Pull request closed without merging**